nvflare/app_common/widgets/log_streaming.py

```diff
@@ -39,6 +39,11 @@ def __init__(self, log_level=""):
         Widget.__init__(self)
         logging.StreamHandler.__init__(self)
         self.log_level = getattr(logging, log_level, logging.INFO)
+        if self.log_level < logging.INFO:
+            raise ValueError(
+                f"LogAnalyticsSender log level must be higher than logging.INFO: {logging.INFO}. "
```

Comment:
The error msg should say that "... higher than DEBUG ...", or " ... higher than or equal to INFO".

Why do we need "{logging.INFO}" in the msg?

Reply:
Added self.log_level initialization enhancement.